addons/pharma/functions/fnc_treatmentAdvanced_Amiodarone.sqf

```diff
@@ -0,0 +1,29 @@
+#include "script_component.hpp"
+/*
+ * Author: MiszczuZPolski
+ * Starts Amiodarone treatment
+ *
+ * Arguments:
+ * 0: Medic <OBJECT>
+ * 1: Patient <OBJECT>
+ * 2: Body Part <STRING>
+ * 3: Treatment <STRING>
+ * 4: Item User (not used) <OBJECT>
+ * 5: Used Item <STRING>
+ *
+ * Return Value:
+ * Succesful treatment started <BOOL>
```

Comment:
where is the return value? is a return Value needed? If not only change the comment :)